tests/functional/adapter/custom_schema_tests/test_custom_database.py

```diff
@@ -0,0 +1,111 @@
+import pytest
+import os
+from dbt.tests.util import (
+    check_relations_equal,
+    check_table_does_exist,
+    run_dbt
+)
+from tests.functional.adapter.custom_schema_tests.seeds import seed_queries
+
+
+_VIEW_1_SQL = """
+select * from {{ target.schema }}.seed
```

Comment:
Since this is creating the thing that you're testing for, I would use the built in ref() function:
```
select * from ref('seed')
```

Reply:
This fails the test. Just tried it.

Reply:
Apologies
```
select * from {{ ref('seed') }}
```

Reply:
Yep, that's what I did. It fails.